examples/api/custom_scale_example.py

```diff
@@ -138,6 +138,7 @@ class InvertedMercatorLatitudeTransform(mtransforms.Transform):
         input_dims = 1
         output_dims = 1
         is_separable = True
+        has_inverse = True
 
```

Comment:
I've fixed this and the file appears clear upon running pep8.  Is there something else I'm missing?